src/test/java/user/UserDaoTest.java

```diff
@@ -0,0 +1,68 @@
+package user;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UserDaoTest {
```

Comment:
환경에 따라 실패할 수 도 있는 테스트입니다 (UserDao가 localhost 의존)
테스트가 실패하면 배포 자체가 안 되는 팀이 많습니다. 
임시로 처리해줍시다

```suggestion
@Disabled("localhost 환경에 의존적이므로 비활성화")
class UserDaoTest {
```

Reply:
테스트 자바 학습 테스트(?) 를 진행하며 @Disabled 을 쓸데가 있나 싶었는데 이런 경우에 쓸 수 있겠네요.